pkg/xds/bootstrap/generator_test.go

```diff
@@ -235,59 +235,6 @@ var _ = Describe("bootstrapGenerator", func() {
 			expectedConfigFile: "generator.custom-config-minimal-request-and-delta.golden.yaml",
 			hdsEnabled:         true,
 		}),
-		Entry("custom config", testCase{
-			dpAuthForProxyType: authEnabled,
-			serverConfig: func() *bootstrap_config.BootstrapServerConfig {
-				return &bootstrap_config.BootstrapServerConfig{
-					Params: &bootstrap_config.BootstrapParamsConfig{
-						AdminAddress:       "192.168.0.1", // by default, Envoy Admin interface should listen on loopback address
-						AdminPort:          9902,          // by default, turn off Admin interface of Envoy
-						AdminAccessLogPath: os.DevNull,
-						XdsHost:            "localhost",
-						XdsPort:            15678,
-						XdsConnectTimeout:  config_types.Duration{Duration: 2 * time.Second},
-					},
-				}
-			}(),
-			dataplane: defaultDataplane,
-			request: types.BootstrapRequest{
-				Mesh:            "mesh",
-				Name:            "name.namespace",
-				DataplaneToken:  "token",
-				OperatingSystem: "windows",
```

Comment:
it seems like nothing else checks `DataplaneResource:` so I'd just change the OS here and not remove the whole test case

Reply:
The golden file is impacted: https://github.com/kumahq/kuma/pull/13679/files#diff-6f4f8f06bb39fa49da2d1e2097397ca56c255bec0cff314e68f7baf7b07fe057L5-L6

In linux it's `/dev/null`, in windows it's `NUL`.